lib/argument_parser.js

```diff
@@ -78,8 +78,9 @@ function ArgumentParser(options) {
 
   this.formatterClass = (options.formatterClass || HelpFormatter);
   this.fromfilePrefixChars = options.fromfilePrefixChars || null;
-  this._positionals = this.addArgumentGroup({ title: 'Positional arguments' });
-  this._optionals = this.addArgumentGroup({ title: 'Optional arguments' });
+  this._positionals = this.addArgumentGroup({ title: 'positional arguments' });
+  this._requireds = this.addArgumentGroup({ title: 'required arguments' });
```

Comment:
https://github.com/python/cpython/blob/2bd37c227e8a042e036c7455d974e3d0b36aed53/Lib/argparse.py#L1649 - can't find appropriate line in mainstream